src/Components/ViewEvent.jsx

```diff
@@ -153,7 +153,7 @@ class ViewEvent extends Component {
               <Col xs={12} lg={6}>
                 <Form.Group controlId="meetingLinkFormGroup">
                   <Form.Label className="rowStyle">Meeting Link:</Form.Label>
-                  <Form.Control type="text" name="meetingLinkInput" disabled={this.state.disabled} value={this.state.meetingLink} onChange={this.changeMeetingLink} />
+                  <Form.Control type="text" name="meetingLinkInput" disabled="true" value={this.state.meetingLink} onChange={this.changeMeetingLink} />
```

Comment:
just a thought do we want to change this from a form to just a link on the screen since we are not allowing the user to edit it anymore?

Reply:
Just let me know what to change it to or feel free to directly push to the branch